InformationScripting/src/handlers/HQuery.cpp

```diff
@@ -233,11 +233,12 @@ bool HQuery::canBeRemoved(const QString& exp, int index)
 		if (index > 0) before = exp[index-1];
 		if (index < exp.length()-1) after = exp[index+1];
 
+		// Since all operators start with OP_PIPE it is enough to check for the pipe below:
```

Comment:
To make this true, we should probably remove the `OperatorQueryNode::OperatorTypes::Union, "U"` above.